src/include/security/helps.rb

```diff
@@ -352,9 +331,53 @@ def initialize_security_helps(include_target)
         "EXTRA_SERVICES"                            => _(
           "<P>Every running service is a potential target of a security attack. Therefore it is recommended to turn off all services which are not used by the system.</P>"
         )
-      } 
+      }
 
+    end
+
+    def boot_dialog_help
+      help = _(
+        "<p><b><big>Boot Security</big></b></p>\n<p>In this dialog, change various boot settings related to security.</p>"
+      )
+
+      if ::Security::CtrlAltDelConfig.default == "reboot"
+        # TRANSLATORS: part of help text - default action (the default is
+        # reboot)
+        details = _(
+          "Usually the system reboots. Sometimes it is desirable\n" \
+            "to ignore this event, for example, when the system serves as both\n" \
+            "workstation and server.</p>"
+        )
+      else
+        # TRANSLATORS: part of help text - default action (the default is halt)
+        details = _(
+          "By default the system halts but sometimes it is desirable\n" \
+            "to ignore this event, for example, when the system serves as both\n" \
+            "workstation and server.</p>"
+        )
+      end
+
+      # Boot dialog help 2/4
+      # TRANSLATORS: %s is help text - default action
+      help += _(
+        "<p><b>Interpretation of Ctrl + Alt + Del</b>:\n" \
+          "Configure what the system should do in response to\n" \
+          "someone at the console pressing the CTRL + ALT + DEL key\n" \
+          "combination. %s"
```

Comment:
Please move the closing `</p>` tag here from the `details` message. It is difficult to verify that the `<p>`... `</p>` pair is complete. And some translators could be confused and would be tempted to add "missing" tags.